javascript-loops-practice/get-student-names.js

```diff
@@ -0,0 +1,11 @@
+/* exported getStudentNames */
+function getStudentNames(students) {
+  const names = [];
+  for (let i = 0; i < students.length; i++) {
+    const obj = students[i];
+    for (const key in obj) {
+      names.push(obj[key]);
+    }
```

Comment:
No `for..in` loop needed!. You can just do `students[i].name`. You want to avoid nesting loops as much as possible, because it increases the complexity & runtime of our code